### PR TITLE
Mne launch

### PR DESCRIPTION
**Pull request by chdinh**
A new MNE GUI hub to have everything in one place

### CODE REVIEW

File: applications/mne_launch/application.cpp [hunk 1]
```diff
@@ -0,0 +1,87 @@
+//=============================================================================================================
+/**
+* @file     application.cpp
+* @author   Christoph Dinh <chdinh@nmr.mgh.harvard.edu>
```
Comment (LorenzE) [inline applications/mne_launch/application.cpp]: Matti as co author? Please check all files in this PR

File: applications/mne_launch/mne_launch.pro [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+# @file     mne_launch.pro
+# @author   Christoph Dinh <chdinh@nmr.mgh.harvard.edu>;
+#           Lorenz Esch <lorenz.esch@tu-ilmenau.de>;
+#           Matti Hamalainen <msh@nmr.mgh.harvard.edu>
+# @version  1.0
+# @date     November, 2016
+#
+# @section  LICENSE
+#
+# Copyright (C) 2013, Christoph Dinh, Lorenz Esch and Matti Hamalainen. All rights reserved.
```
Comment (LorenzE) [inline applications/mne_launch/mne_launch.pro]: update year

File: applications/mne_launch/mnelaunchcontrol.h [hunk 3]
```diff
@@ -0,0 +1,116 @@
+//=============================================================================================================
+/**
+* @file     mnelaunchcontrol.cpp
```
Comment (LorenzE) [inline applications/mne_launch/mnelaunchcontrol.h]: change to .h

File: applications/mne_launch/qml/ApplicationButton.qml [hunk 4]
```diff
@@ -0,0 +1,118 @@
+//=============================================================================================================
+/**
+* @file     ApplicationButton.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/ApplicationButton.qml]: lowercased file name?

File: applications/mne_launch/qml/CenterDot.qml [hunk 5]
```diff
@@ -0,0 +1,89 @@
+//=============================================================================================================
+/**
+* @file     CenterDot.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/CenterDot.qml]: lowercased file name?

File: applications/mne_launch/qml/ConnectorLine.qml [hunk 6]
```diff
@@ -0,0 +1,83 @@
+//=============================================================================================================
+/**
+* @file     ConnectorLine.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/ConnectorLine.qml]: lowercased file name?

File: applications/mne_launch/qml/Logo.qml [hunk 7]
```diff
@@ -0,0 +1,77 @@
+//=============================================================================================================
+/**
+* @file     Logo.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/Logo.qml]: lowercased file name?

File: applications/mne_launch/qml/MainPage.qml [hunk 8]
```diff
@@ -0,0 +1,97 @@
+//=============================================================================================================
+/**
+* @file     MainPage.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/MainPage.qml]: lowercased file name?

File: applications/mne_launch/qml/MainPageForm.ui.qml [hunk 9]
```diff
@@ -0,0 +1,258 @@
+//=============================================================================================================
+/**
+* @file     MainPageForm.ui.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/MainPageForm.ui.qml]: lowercased file name?

File: applications/mne_launch/qml/MenuButton.qml [hunk 10]
```diff
@@ -0,0 +1,107 @@
+//=============================================================================================================
+/**
+* @file     MenuButton.qml
+* @author   Christoph Dinh <chdinh@nmr.mgh.harvard.edu>
```
Comment (LorenzE) [inline applications/mne_launch/qml/MenuButton.qml]: lowercased file name?

File: applications/mne_launch/qml/SampleDataStatus.qml [hunk 11]
```diff
@@ -0,0 +1,93 @@
+//=============================================================================================================
+/**
+* @file     SampleDataStatus.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/SampleDataStatus.qml]: lowercased file name?

File: applications/mne_launch/qml/main.qml [hunk 12]
```diff
@@ -0,0 +1,59 @@
+//=============================================================================================================
+/**
+* @file     main.qml
```
Comment (LorenzE) [inline applications/mne_launch/qml/main.qml]: lowercased file name?

**Comment by chdinh** [comment]
QML file names are an exception to lower case filenames, since they have to match the Components Naming used in the Source Code itself. Qt is also naming qml files with an Upper Case letter

**Pull request merged**